docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/soa-applications.md

```diff
@@ -3,15 +3,17 @@ title: SOA applications
 description: Bear in mind that containers can also be a useful deployment option for SOA applications.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 11/23/2018
+ms.date: 02/15/2019
 ---
-# SOA applications
+# Service-oriented applications
 
-SOA was an overused term and meant so many different things to different people. But at a minimum and as a common denominator, SOA, or service orientation, mean that you structure the architecture of your application by decomposing it in multiple services (most commonly as HTTP services) that can be classified in different types like subsystems or, in other cases, as tiers.
+Service-Oriented Architecture (SOA) was an overused term and meant so many different things to different people. But, as a common denominator, SOA means that you structure the architecture of your application by decomposing it in several services (most commonly as HTTP services) that can be classified in different types like subsystems or, in other cases, as tiers.
 
-Today, you can deploy those services as Docker containers, which solves deployment-related issues because all of the dependencies are included within the container image. However, when you need to scale-out SOAs, you might encounter challenges if you are deploying based on single instances. This is where a Docker clustering software or orchestrator will help you. We'll look at this in greater detail in the next section when we examine microservices approaches.
+Today, you can deploy those services as Docker containers, which solve deployment-related issues because all of the dependencies are included in the container image. However, when you need to scale out SOAs, you might encounter challenges if you are deploying based on single instances. This challenge can be handled using a Docker clustering software or orchestrator. We'll look at orchestrators in greater detail in the next section, when we explore microservices approaches.
```

Comment:
```suggestion
Today, you can deploy those services as Docker containers, which solve deployment-related issues because all of the dependencies are included in the container image. However, when you need to scale out SOAs, you might encounter challenges if you are deploying based on single instances. This challenge can be handled using Docker clustering software or an orchestrator. We'll look at orchestrators in greater detail in the next section, when we explore microservices approaches.
```